ovos_utils/configuration.py

```diff
@@ -17,14 +18,13 @@
 
 def get_xdg_config_dirs(folder=None):
     folder = folder or get_xdg_base()
-    paths = xdg_config_dirs()
-    paths.append(xdg_config_home())
-    return [join(path, folder) for path in paths if isdir(join(path, folder))]
+    xdg_dirs = xdg_config_dirs() + [xdg_config_home()]
+    return [join(path, folder) for path in xdg_dirs]
 
 
 def get_xdg_data_dirs(folder=None):
     folder = folder or get_xdg_base()
-    return [join(path, folder) for path in xdg_data_dirs() if isdir(join(path, folder))]
```

Comment:
isdir check removed because these paths can still be consumed and created, some downstream tools apparently broke because they expected this behavior